examples/in-app-message-example/src/App.tsx

```diff
@@ -135,7 +69,8 @@ function App() {
         <>
           <h1>Knock In-App Message Example</h1>
           <hr />
-          <Message />
+          <h2>Banner</h2>
+          <Banner.Default />
```

Comment:
I would *really* like it if this was `<Banner />`. That might mean that we need to have an export like:

`import { Banner } from @knocklabs/react/in-app/banner`

Reply:
Yeah, left a todo to clean that up. Will sync with Kyle to think through how we can scope everything based on how we normally export the grouped components